### PR TITLE
[Android] Add includeReferrer to allow adding android package name as referrer for website to track

### LINKED ISSUE #86: Add your app as the referrer (Android)
Check here https://developer.chrome.com/multidevice/android/customtabs#add-your%20app%20as%20the%20referrer

### PR DESCRIPTION
**Pull request by vincent-paing**
## What is the current behavior?

There is no way to add referrer intent extra in android.

## What is the new behavior?

Added `includeReferrer` options for Android to add your package name as referrer when launching in in app browser:

See: https://developer.chrome.com/docs/android/custom-tabs/best-practices/#add-your-app-as-the-referrer

Fixes #86

### CODE REVIEW

**Comment by jdnichollsc** [comment]
Thanks for your amazing contribution mate! <3

File: index.d.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ declare module 'react-native-inappbrowser-reborn' {
     hasBackButton?: boolean,
     browserPackage?: string,
     showInRecents?: boolean
```
Comment (jdnichollsc) [inline index.d.ts:60]: @vincent-paing Oh a comma is missing here, I didn't see that! 😮

Comment (vincent-paing) [reply index.d.ts:60]: Ah, my bad. Have you patched this?

Comment (jdnichollsc) [reply index.d.ts:60]: Sure thing, I used Prettier to fix the format of this file, thanks!

**Pull request merged**